articles/article-063-reverse-engineering.md

```diff
@@ -0,0 +1,26 @@
+<!-- BEGIN TITLE -->
+# Reverse Engineering
+<!-- END TITLE -->
+
+<!-- BEGIN BODY -->
+In general terms, *Reverse Engineering* describes the process of analyzing an existing apparatus to figure out how it works and how it was made.
+The same can be done for a computer program but instead of looking at screws and cogwheels we are looking at the machine instructions.
+These are effectively the 0's and 1's that tell the computer what to do.
+This is much harder to understand than the code that was used to generate the program.
```

Comment:
As we are stating here the obvious, we could say 
"The machine code is obviously much harder to understand, ..."